libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -477,6 +477,73 @@ bool AP_Mission::set_current_cmd(uint16_t index)
     return true;
 }
 
+// returns false on any issue at all.
+bool AP_Mission::set_item(uint16_t index, mavlink_mission_item_int_t& src_packet)
+{
+        // this is the on-storage format
+        AP_Mission::Mission_Command cmd;
+
+        // can't handle request for anything bigger than the mission size+1...
+        if (index > num_commands() ) {  return false; }
```

Comment:
normally we would put the return false on a new line.  it's a small thing so not a blocker.

Reply:
ran thru astyle, fixed.